CIP-0030/README.md

```diff
@@ -1,72 +1,93 @@
 ---
 CIP: 30
 Title: Cardano dApp-Wallet Web Bridge
-Authors: rooooooooob
-Comments-URI: https://github.com/cardano-foundation/CIPs/pull/88
-Status: Draft
-Type: Standards
+Status: Active
+Category: Wallets
+Authors:
+  - rooooooooob
+Implementors:
+  - Begin <https://begin.is/>
+  - Eternl <https://eternl.io/>
+  - Flint <https://flint-wallet.com/>
+  - GeroWallet <https://www.gerowallet.io/>
+  - Lace <https://www.lace.io/>
+  - Nami <https://namiwallet.io/>
+  - NuFi <https://nu.fi/>
+  - RayWallet <https://raywallet.io/>
+  - Yoroi <https://yoroi-wallet.com/>
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/88
+  - https://github.com/cardano-foundation/CIPs/pull/148
+  - https://github.com/cardano-foundation/CIPs/pull/151
+  - https://github.com/cardano-foundation/CIPs/pull/183
+  - https://github.com/cardano-foundation/CIPs/pull/208
+  - https://github.com/cardano-foundation/CIPs/pull/323
+  - https://github.com/cardano-foundation/CIPs/issues/169
+  - https://github.com/cardano-foundation/CIPs/issues/178
+  - https://github.com/cardano-foundation/CIPs/issues/204
+  - https://github.com/cardano-foundation/CIPs/issues/253
+  - https://github.com/cardano-foundation/CIPs/issues/386
+  - https://github.com/cardano-foundation/CIPs/issues/404
+  - https://github.com/cardano-foundation/CIPs/issues/419
+  - https://github.com/cardano-foundation/CIPs/pull/446
 Created: 2021-04-29
 License: CC-BY-4.0
 ---
 
-# Abstract
+## Abstract
 
 This documents describes a webpage-based communication bridge allowing webpages (i.e. dApps) to interface with Cardano wallets. This is done via injected javascript code into webpages. This specification defines the manner that such code is to be accessed by the webpage/dApp, as well as defining the API for dApps to communicate with the user's wallet. This document currently concerns the Shelley-Mary era but will have a second version once Plutus is supported. This specification is intended to cover similar use cases as web3 for Ethereum or [EIP-0012](https://github.com/ergoplatform/eips/pull/23) for Ergo. The design of this spec was based on the latter.
 
-
-# Motivation
+## Motivation: why is this CIP necessary?
 
 In order to facilitate future dApp development, we will need a way for dApps to communicate with the user's wallet. While Cardano does not yet support smart contracts, there are still various use cases for this, such as NFT management. This will also lay the groundwork for an updated version of the spec once the Alonzo hardfork is released which can extend it to allow for Plutus support.
 
+## Specification
 
+### Data Types
 
-# Specification
-
-## Version
-
-The API specified in this document will count as version 0.1.0 for version-checking purposes below.
+#### Address
 
-## Data Types
+A string representing an address in either bech32 format, or hex-encoded bytes. All return types containing `Address` must return the hex-encoded bytes format, but must accept either format for inputs.
```

Comment:
Why did this PR change the address format for return types from bech32 to hex-encoded? Seems unrelated to this PR

Reply:
according to how it's "implemented in practice": https://github.com/cardano-foundation/CIPs/pull/446#discussion_r1102723847

Reply:
Because it was indicated that this is how most wallets have implemented it today. And the goal was to reflect on the current reality.